ui/src/components/Locations/LocationsMinionCmd.vue

```diff
@@ -15,7 +15,7 @@
         Copy
       </template>
     </FeatherButton>
-    <p>Please download the certificate and replace the pathToFile in the command below.</p>
+    <p>Make sure you replace <strong>PATH_TO_DOWNLOADED_FILE</strong> with the path to the certificate file.</p>
```

Comment:
```suggestion
    <p>Make sure you replace <strong>PATH_TO_DOWNLOADED_FILE</strong> with the full path to the certificate file.</p>
```
This was a comment that came up in the user testing last week, so adding it here.